### PR TITLE
Hugo: add spotlight shortcode

### PR DESCRIPTION
**Pull request by anne-usmedia**
- Added spotlight example page (/examples/shortcodes/spotlight/)
- Added link on example overview (/examples/)
- Added Related content example to blog example page (/blog/example/)
- Set up spotlight shortcode
- Added spotlight styling

For https://linear.app/usmedia/issue/CUE-308

### CODE REVIEW

File: content/blog/example/en.md [hunk 1]
```diff
@@ -13,8 +13,8 @@ tags:
     - Ecosystem
     - cue command
 meta:
-  - type: readingTime
-    value: 5
+    - type: readingTime
```
Comment (JorindeUsMedia) [inline content/blog/example/en.md:16]: is this indenting change correct?

File: hugo/assets/scss/components/highlight.scss [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        padding-bottom: 1.5rem;
+    }
+
+    &__content {
+        --heading-color: #{ $c-grey--darkest };
+        --text-link-color: #{ $c-grey--darkest};
+
+        color: var(--highlight-text-color, $c-grey--darkest);
+
+        :first-child {
```
Comment (JorindeUsMedia) [inline hugo/assets/scss/components/highlight.scss:32]: maybe use the marginless-child mixin here?

**Review by JorindeUsMedia — CHANGES_REQUESTED** [review]
Looks good :) Just some comments.
And I'm not sure how but can we somehow improve the anchors positioning within the highlight?
Screenshot from 2023-08-25 15-09-39

File: content/blog/example/en.md (hunk 1 above)
Comment (anne-usmedia) [reply content/blog/example/en.md:16]: yes

**Review by JorindeUsMedia — APPROVED** [review]

**Comment by myitcv** [comment]
Looks like there is some naming inconsistency here: are we going for `spotlight` or `highlight`? The commit message, example, and actual code all seem to disagree :)

Please can we also rebase this PR?

**Comment by JorindeUsMedia** [comment]
> Looks like there is some naming inconsistency here: are we going for `spotlight` or `highlight`? The commit message, example, and actual code all seem to disagree :)
> 
> Please can we also rebase this PR?

We went for 'spotlight' since 'highlight' was already reserved by hugo's code highlighting https://gohugo.io/content-management/syntax-highlighting/#highlight-shortcode

**Review by myitcv — APPROVED** [review]
LGTM - will import into GerritHub now, and report a different issue I've seen via Linear

**Comment by jpluscplusm** [comment]
Closed via 3efff44abf7b59e152453db607fa9fa159c914cc

**Pull request closed without merging**